airbyte-integrations/connectors/source-sf-marketingcloud-singer/build.gradle

```diff
@@ -0,0 +1,14 @@
+plugins {
+    id 'airbyte-python'
+    id 'airbyte-docker'
+    id 'airbyte-source-acceptance-test'
+}
+
+airbytePython {
+    moduleDirectory 'source_sf_marketingcloud_singer'
+}
+
+dependencies {
+    implementation files(project(':airbyte-integrations:bases:source-acceptance-test').airbyteDocker.outputs)
+    implementation files(project(':airbyte-integrations:bases:base-singer').airbyteDocker.outputs)
```

Comment:
I don't think we need this line anymore since we using python3.7-slim as a base image